### PR TITLE
[ADD] - add contract_queue_job and contract_invoice_auto_validate

### PR DESCRIPTION
**Pull request by sbejaoui**

### CODE REVIEW

File: contract_queue_job/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "name": "Contract Queue Job",
+    "summary": """
+        This addon make contract invoicing cron plan each contract in a job
+        instead of creating all invoices in one transaction""",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,"
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/contract",
+    "depends": ['contract'],
```
Comment (houssine78) [inline contract_queue_job/__manifest__.py]: are we not missing a dependency on queue_job ?

Comment (sbejaoui) [reply contract_queue_job/__manifest__.py]: yes, I fixed it

Comment (houssine78) [reply contract_queue_job/__manifest__.py]: great ! all LGTM now !

**Review by houssine78 — APPROVED** [review]

**Review by cesarlr — APPROVED** [review]

**Comment by sbejaoui** [comment]
I added some unit tests

**Pull request merged**